cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -21,23 +21,40 @@ function error_exit () {
   script=`basename $0`
   echo "parallelcluster: ${script} - $1"
   logger -t parallelcluster "${script} - $1"
-  echo "$1. If --rollback-on-failure was set to false, check /var/log/cfn-init.log and /var/log/cloud-init-output.log for more details." > /var/log/parallelcluster/bootstrap_error_msg
+  case $custom_script in
+    OnNodeStart|OnNodeConfigured)
```

Comment:
we have OnNodeUpdated now in develop. Be sure to add it too.

Reply:
Thanks for the reminder. I skipped OnNodeUpdated here in purpose because it is out of scope of the corresponding design. We haven't done a thorough investigation on cluster update errors and designed error code/messages yet.